### PR TITLE
215 reservation logic rev:deletion invalids characters

### PR DESCRIPTION
**Pull request by ebercamargo**
#215 reservation logic rev:deletion invalids characters

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#401](https://github.com/timelesslounge/timelessis/issues/401) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#401](https://github.com/timelesslounge/timelessis/issues/401) is assigned to @vryazanov[/z](https://www.0crat.com/u/vryazanov), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/e0cd43c4-c690-4634-8f93-925b3643d0ec); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: timeless/reservations/views.py [hunk 1]
```diff
@@ -94,19 +94,22 @@ def get(self, company_id):
...
 def list_reservations(reservations):
-    """
-        @todo #172:30min Refactor this after the implementation of GenericViews.
-         Take a look at puzzles #134 and #173 where the requirements of generic
-         views are described. Don't forget to cover the generated code with
-         tests
-
-    :param reservations:
-    :return:
+    reservations_result = []
```
Comment (vryazanov) [inline timeless/reservations/views.py]: @ebercamargo check `ListView` in `timeless.view` and use it

**Comment by 0crat** [comment]
@vryazanov[/z](https://www.0crat.com/u/vryazanov) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by emilianodellacasa** [comment]
@ebercamargo closed for pr #463

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @vryazanov[/z](https://www.0crat.com/u/vryazanov), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#401](https://github.com/timelesslounge/timelessis/issues/401) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@vryazanov 
According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by vryazanov** [comment]
@ypshenychka  I confirm

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Quality review completed: +6 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @vryazanov[/z](https://www.0crat.com/u/vryazanov)

**Pull request closed without merging**